client/app/lib/providers/computecontroller.coffee

```diff
@@ -78,6 +84,17 @@ module.exports = class ComputeController extends KDController
         @info machine for machine in @machines
 
 
+  bindGroupStatusEvents: ->
+
+    { groupsController } = kd.singletons
+
+    # /cc @cihangir: not sure if this is the right way to bind the event.
+    groupsController.on 'payment_status_changed', ({ oldStatus, newStatus }) =>
+      wasDisabled = @disabled
+      @disabled = isGroupDisabled getGroup()
+
+      do @reset  unless @disabled isnt wasDisabled
```

Comment:
`do @reset  if @disabled is wasDisabled` would be more logically logical maybe :)